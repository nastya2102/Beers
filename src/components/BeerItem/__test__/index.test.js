import React from "react";
import { render } from "@testing-library/react";
import BeerItem from "../index";

describe("BeerItem", () => {
  test("should render BeerItem component", () => {
    const { container } = render(
      <BeerItem
        name={true}
        image_url={"//"}
        tagline={<span>123</span>}
        abv={<span>457</span>}
        description={<span>457</span>}
        first_brewed={<span>457</span>}
        brewers_tips={<span>457</span>}
      />
    );

    expect(container).toBeTruthy();
  });

  test("should match snapshot BeerItem component", () => {
    const { container } = render(
      <BeerItem
        name={true}
        image_url={"//"}
        tagline={<span>123</span>}
        abv={<span>457</span>}
        description={<span>457</span>}
        first_brewed={<span>457</span>}
        brewers_tips={<span>457</span>}
      />
    );

    expect(container).toMatchSnapshot();
  });
});
