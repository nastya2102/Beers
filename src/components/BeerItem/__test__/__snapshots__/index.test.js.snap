// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BeerItem should match snapshot BeerItem component 1`] = `
<div>
  <div
    class="sc-eCssSg VkFkB"
  >
    <div
      class="sc-jSgupP bzoAXb"
    >
      <div
        class="sc-iBPRYJ gpjgM"
      >
        <span>
          457
        </span>
      </div>
      <div
        class="sc-gsTCUz esqFhx"
        height="500px"
      />
    </div>
    <div
      class="sc-gKsewC dcdIQI"
    >
      <div
        class="sc-bdfBwQ tXWSF"
      />
      <div
        class="sc-kEjbxe gFLfWc"
      >
        Since 
        <span>
          457
        </span>
      </div>
      <div
        class="sc-fubCfw bmDaIS"
      >
        <span>
          123
        </span>
      </div>
      <div
        class="sc-pFZIQ jIGNkE"
      >
        <span>
          457
        </span>
      </div>
      <div
        class="sc-pFZIQ jIGNkE"
      >
        <span
          class="sc-jrAGrp jaUFKk"
        >
          Brewers tips: 
        </span>
        <span>
          457
        </span>
      </div>
    </div>
  </div>
</div>
`;
